manager/controlapi/hackpicker/raftpicker.go

```diff
@@ -0,0 +1,138 @@
+package hackpicker
```

Comment:
Let's add a comment here about why this package still exists and what will be done about it.